src/openms/source/QC/DBSuitability.cpp

```diff
@@ -290,9 +229,320 @@ namespace OpenMS
     return true;
   }
 
-  bool DBSuitability::passesFDR_(const PeptideHit& hit, double FDR)
+  bool DBSuitability::passesFDR_(const PeptideHit& hit, double FDR) const
   {
     if (hit.getScore() > FDR) return false;
     return true;
   }
-}
+
+  pair<String, Param> DBSuitability::extractSearchAdapterInfoFromMetaValues_(const ProteinIdentification::SearchParameters& search_params) const
+  {
+    Param p;
+    // list of all allowed adapters
+    vector<String> working_adapters{ "CometAdapter", "CruxAdapter", "MSGFPlusAdapter", "MSFraggerAdapter", "MyriMatchAdapter", "OMSSAAdapter", "XTandemAdapter" };
```

Comment:
would automatic extraction using all names from ToolHandler that end in "Adapter" also work?

Reply:
Dont think so, e.g. FidoAdapter and a few others do not do DB search